Scripts/build-xcframework.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Work dir is directory where all XCFramework artifacts is stored.
+WORK_DIR="${SRCROOT}/build"
```

Comment:
Avoid dependency to `SRCROOT`, use the same folder like in other places. There are built in vars for it.